### ISSUE TITLE
Unexpanded variable in configure help output

### ISSUE THREAD

**Issue by ryandesign** [opencog/link-grammar#1491]
With link-grammar 5.12.4 when I run `./configure --help` on macOS I see this output:

```
Optional Packages:
```
```
  --with-hunspell-dictdir=DIR
                          Use DIR to find HunSpell files
                          (default=$default_hunspell_dictdir
```

There's a missing closing parenthesis, but more importantly the value of `$default_hunspell_dictdir` hasn't been expanded.

**Comment by ampli** [comment]
I addressed it in PR #1494.

**Comment by linas** [comment]
#1494 was merged so I'm closing this
